### PR TITLE
Allow the GenericFile show page to be loaded exclusively from solr

### PR DESCRIPTION
**Pull request by cam156**
Needed to adjust Audits and Characterization slightly to accommodate the solr backed GenericFile not having access to the same api as the fedora backed GenericFile.

### CODE REVIEW

File: app/controllers/concerns/sufia/files_controller_behavior.rb [hunk 1]
```diff
@@ -39,13 +39,20 @@ module FilesControllerBehavior
...
+      authorize_resource only: [:show]
 
       class_attribute :edit_form_class, :presenter_class
       self.edit_form_class = Sufia::Forms::GenericFileEditForm
       self.presenter_class = Sufia::GenericFilePresenter
     end
 
+    def load_resource_from_solr
+      @generic_file = ::GenericFile.load_instance_from_solr(params[:id])
+      @generic_file
```
Comment (jcoyne) [inline app/controllers/concerns/sufia/files_controller_behavior.rb]: This line can be omitted I think.

**Comment by jcoyne** [comment]
Should there be a test for the show view?

**Comment by cam156** [comment]
@jcoyne I was not changing the view, and made sure the controller did not find the GenericFile.  There is already a test for the show view here: https://github.com/projecthydra/sufia/blob/master/spec/views/generic_file/show.html.erb_spec.rb
Is there additional test that you are thinking about?

**Comment by jcoyne** [comment]
@cam156 Ah, I see. You are still returning a GenericFile object, just one loaded from solr. I was thinking you were returning a SolrDocument. So never mind.

**Pull request merged**